### PR TITLE
Build any pull request

### PR DESCRIPTION
**Pull request by hannesa2**
Now pull requests with a target of an other pull request will build too

### CODE REVIEW

File: .github/workflows/Android-CI.yml [hunk 1]
```diff
@@ -5,12 +5,10 @@ on:
     branches:
       - master
   pull_request:
-    branches:
-      - master
 
 jobs:
   buildTest:
-    name: Build & Test JDK ${{ matrix.java_version }}
```
Comment (hannesa2) [inline .github/workflows/Android-CI.yml:13]: There is no more matrix at all

**Comment by hannesa2** [comment]
The remaining question is from where this `11.0.3` comes from ?

image

**Comment by hannesa2** [comment]
When you go to action overview, all is fine https://github.com/niqdev/ipcam-view/actions/workflows/Android-CI.yml

I guess there is something hanging, because on earlier days (eg. ba1278e7305) there was something like this 
image

**Comment by hannesa2** [comment]
See https://github.com/niqdev/ipcam-view/pull/172

File: .github/workflows/Android-CI.yml [hunk 2]
```diff
@@ -5,12 +5,10 @@ on:
     branches:
       - master
   pull_request:
-    branches:
-      - master
```
Comment (niqdev) [inline .github/workflows/Android-CI.yml:9]: do we want to allow that? what would be the use case?

**Pull request closed without merging**